src/components/LoginSignup/LoginSignupPopup.jsx

```diff
@@ -62,7 +60,27 @@ const LoginSignupPopup = () => {
 
     return (
         <>
-            <Button onClick={onOpen}>Login/Signup</Button>
+            <Button
+                display={{ base: 'none', sm: 'flex' }}
+                onClick={() => {
+                    setMode('login');
+                    onOpen();
+                }}
+                variant="outline"
```

Comment:
Why did you style the buttons differently? I think it'd look better if they were styled the same.
Otherwise everything looks great.

Reply:
I was inspired by github's:
<img width="505" alt="Screen Shot 2021-08-06 at 2 10 12 PM" src="https://user-images.githubusercontent.com/43733599/128571384-5b1849ff-9f0d-4f53-875a-f891f9eb82d2.png">

Reply:
I see, I guess they wanna draw attention to sign up for new users.

In that case, what about fully mimicking their styling? So we'd move lines 69-70 into the Sign Up button's styling, and make Log In's styling: `variant="ghost"`

Quite subjective, though I think I might prefer the more Github style one.

_current_
![1_button](https://user-images.githubusercontent.com/51832617/128574609-854151f5-6430-4dcd-9536-410a9aa626e7.png)

_more like github_
![2_button-edit](https://user-images.githubusercontent.com/51832617/128574623-1aa0fa4f-23e0-4b48-8057-3f8c9e7f9c6d.png)

Reply:
Maybe have it more like GitHub? 'Log in' and 'Sign up'